goad

```diff
@@ -52,8 +52,9 @@ else
 		git submodule update --init
 		;;
 	test)
-		go test -i "$SUBSECTION" &&
-		go test -v "$SUBSECTION" && {
+		set +e ; shift ; shift ; set -e
```

Comment:
Yeah.  Bash.  *hands raised in supplication*

Amusing to note that I tried the `||true` method first, but that doesn't work here -- it implicitly spins up a subshell, meaning it doesn't mutate the right args array.  Whee.